Description: Do not assign locale name based on AccountsService's "Language" key to LANG.
Author: Gunnar Hjalmarsson <ubuntu@gunnar.cc>
Bug-Ubuntu: https://launchpad.net/bugs/864618

Index: lightdm/src/session.c
===================================================================
--- lightdm.orig/src/session.c	2011-10-05 09:43:39.178108000 +0200
+++ lightdm/src/session.c	2011-10-05 09:46:52.671534207 +0200
@@ -471,7 +471,7 @@
     /* Do PAM actions requiring session process */
     pam_session_setup (session->priv->authentication);
     set_env_from_authentication (session, session->priv->authentication);
-    set_language (session);
+    /* set_language (session); Disabled, see LP#864618 */
 
     PROCESS_CLASS (session_parent_class)->run (process);
 }
